include/tester/ssp/Softphone.hpp

```diff
@@ -1,15 +1,72 @@
 #pragma once
 
+#include<iostream>
+#include<string>
+#include<functional>
+#include<memory>
+
 #include"SSPAccount.hpp"
 #include"SSPCall.hpp"
+#include"SoftphoneArguments.hpp"
 
 class Softphone
 {
 public:
-    Softphone(/* args */);
-    ~Softphone();
+    Softphone(const SoftphoneArguments & args):
+        _account(args.id, args.domain, args.secret, std::bind(&Softphone::onIncomingCall, this, std::placeholders::_1)),
+        _call(std::make_shared<SSPCall>(&_account, std::bind(&Softphone::onCallState, this,
+            std::placeholders::_1, std::placeholders::_2)))
+    {
+        _account.apply();
+    }
 
-private:
-    std::vector<SSPAccount *> _accounts;
+    ~Softphone() = default;
+
+    void call(const std::string & uri)
+    {
+        _call->callTo(uri);
+    }
+
+    void onCallState(pj::CallInfo ci, const pj::OnCallStateParam &prm)
```

Comment:
I just looked into this method again, since we can use `_call->getInfo()` anywhere in the `Softphone` class, we don't need the `pj::CallInfo ci` parameter, so remove it and update the code

Reply:
done